tests/unit/fem/test_r1d_bilininteg.cpp

```diff
@@ -0,0 +1,4753 @@
+         {
+            H1_FECollection    fec_h1(order, dim);
+            FiniteElementSpace fespace_h1(&mesh, &fec_h1);
+
+            BilinearForm m_h1(&fespace_h1);
+            m_h1.AddDomainIntegrator(new MassIntegrator());
+            m_h1.Assemble();
+            m_h1.Finalize();
+
+            GridFunction g_h1(&fespace_h1);
+
+            Vector tmp_h1(fespace_h1.GetNDofs());
+
+            SECTION("Without Coefficient")
+            {
+               MixedBilinearForm blf(&fespace_rt, &fespace_h1);
+               blf.AddDomainIntegrator(new MixedScalarDivergenceIntegrator());
+               blf.Assemble();
+               blf.Finalize();
+
+               blf.Mult(f_rt,tmp_h1); g_h1 = 0.0;
+               CG(m_h1, tmp_h1, g_h1, 0, 200, cg_rtol * cg_rtol, 0.0);
+
+               REQUIRE( g_h1.ComputeL2Error(dF3_coef) < tol );
+            }
+            SECTION("With Scalar Coefficient")
+            {
+               MixedBilinearForm blf(&fespace_rt, &fespace_h1);
+               blf.AddDomainIntegrator(
+                  new MixedScalarDivergenceIntegrator(q3_coef));
+               blf.Assemble();
+               blf.Finalize();
+
+               blf.Mult(f_rt,tmp_h1); g_h1 = 0.0;
+               CG(m_h1, tmp_h1, g_h1, 0, 200, cg_rtol * cg_rtol, 0.0);
+
+               REQUIRE( g_h1.ComputeL2Error(qdF3_coef) < tol );
+            }
+         }
+      }
+   }
+}
+
+TEST_CASE("R1D Bilinear Vector Divergence Integrator",
+          "[ND_R1D_FECollection]"
+          "[RT_R1D_FECollection]"
+          "[MixedVectorDivergenceIntegrator]"
+          "[MixedScalarVectorIntegrator]"
+          "[BilinearFormIntegrator]"
+          "[NonlinearFormIntegrator]")
+{
+   int order = 2, n = 1, dim = 1, vdim = 3;
+   double cg_rtol = 1e-14;
+   double tol = 1e-9;
+
+   for (int type = (int)Element::SEGMENT;
+        type <= (int)Element::SEGMENT; type++)
+   {
+      Mesh mesh = Mesh::MakeCartesian1D(n, 2.0);
+
+      VectorFunctionCoefficient   F3_coef(vdim, F3);
+      VectorFunctionCoefficient   V3_coef(vdim, V3);
+      VectorFunctionCoefficient VdF3_coef(vdim, VDivF3);
+
+      SECTION("Operators on RT_R1D for element type " + std::to_string(type))
+      {
+         RT_R1D_FECollection    fec_rt(order - 1, dim);
+         FiniteElementSpace fespace_rt(&mesh, &fec_rt);
+
+         GridFunction f_rt(&fespace_rt); f_rt.ProjectCoefficient(F3_coef);
+
+         REQUIRE( f_rt.ComputeL2Error(F3_coef) < tol );
+
+         SECTION("Divergence of RT_R1D")
+         {
+            DivergenceGridFunctionCoefficient divF_coef(&f_rt);
+
+            L2_FECollection fec_l2(order - 1, dim);
+            FiniteElementSpace fespace_l2(&mesh, &fec_l2);
+
+            GridFunction df_l2(&fespace_l2); df_l2.ProjectCoefficient(divF_coef);
+
+            FunctionCoefficient divF3_coef(DivF3);
+            REQUIRE( df_l2.ComputeL2Error(divF3_coef) < tol );
+         }
+         /*
+              SECTION("Divergence of RT")
```

Comment:
Why is this commented?

Reply:
Good question. I don't see a reason so keep it.